src/fhirpath/search.py

```diff
@@ -351,6 +352,8 @@ def __init__(self, context: SearchContext, query_string=None, params=None):
             self.context.augment_with_types(additional_resource_types)
 
     def build_query_params_string(self, params):
+        # Otherwise, an expection is raised for None values
+        params = {k: v or "" for k, v in params.items()}
```

Comment:
Do we want to keep params that don't have a value when building the url ?

Reply:
Hum, maybe not, indeed!